src/physlearn/names.py

```diff
@@ -0,0 +1,121 @@
+from enum import Enum
+
+
+class DataType(Enum):
+    def __lt__(self, other):
+        return self.value <= other.value
+
+    def __le__(self, other):
+        return self.value < other.value
+
+
+class ModelDataType(DataType):
+    FEATURES = "features"
+
+
+class SignalType(DataType):
+    ECG = "ecg"
+    EEG = "eeg"
+    RRIntervals = "rri"
+    FAKE = "fake signal"
+    Unknown = "unknown"
+    SPO2 = "spo2"
+    HR = "hr"
+
+
+class NonSignalDataType(DataType):
+    HOSPITAL_NAME = "hospital_name"
```

Comment:
1. Some non signal data types here are specific to the epilepsy project such as `NUM_SEIZURES` 
2. Most of the non signal data types are not being used in this repository (they being used by our sources, mostly by the `HdfSource`)

Reply:
We will migrate the sources to here too, after the Apis are in place